docs/FAQ.md

```diff
@@ -1,5 +1,15 @@
 # Frequently asked questions
 
+### Where are my "saves"? I thought `jj` saved the working copy after every edit.
+
+There is a working-copy commit that is updated on every jj command, and it so 
+happens that every commit has an associated change id, so it's true that there 
+is a "change" that's "caused" by running any jj command. You can observe this 
+by looking at the change id, the second highlighted part in `jj log` for a 
+commit, which changes after editing a file or the commit description.
```

Comment:
This second sentence had my brain melting, so you can look at change id.. okay, and then there's the second highlighted part which is the commit that is changing (not change id like the sentence made me think at a first glance).

I guess it should be reworded :upside_down_face:

Reply:
I have trouble understanding the first sentence too. I think maybe you meant s/change id/commit id/ there too.

Reply:
As written, this is confusing to me as well. I'm thinking of what points we'd want to make. Here's a draft:

> Indeed, every `jj` command updates the current "working copy" revision (link to glossary?), marked with `@` in `jj log`. You can notice this by how the *commit id* of the working copy revision changes when it's updated. Note that, unless you move to another revision (with `jj new` or `jj edit`, for example), the *change id* will *not* change.
>
> You can see the history of these changes using `jj obslog`. This will show the history of the commits that were previously the "working copy commit", since the last time the *change id* of the working copy commit changed. The obsolete changes will be marked as "hidden". They are still accessible with any `jj` command (`jj diff`, for example), but you will need to use the *commit id* to refer to hidden commits.
>
> You can also use `jj obslog -r` on revisions that were previously the "working copy" revisions.

Reply:
That draft helped me, but I think there are some holes.
Are these not frequently asked questions? : 
"What are all those hashes?" or "What's the difference between commit ID and change ID?" or "Which IDs are `forever` (or match Git)?"

Reply:
> This second sentence had my brain melting, so you can look at change id.. okay, and then there's the second highlighted part which is the commit that is changing (not change id like the sentence made me think at a first glance).
> 
> I guess it should be reworded 🙃

As soon as I put it out for review, I noticed it too. 

> I have trouble understanding the first sentence too. I think maybe you meant s/change id/commit id/ there too.

My mistake for always mixing these terms.

> That draft helped me, but I think there are some holes.
> Are these not frequently asked questions? :
> "What are all those hashes?" or "What's the difference between commit ID and change ID?" or "Which IDs are `forever` (or match Git)?"

Thanks, but there's a [Glossary](https://github.com/martinvonz/jj/blob/main/docs/glossary.md) which answers your other questions.

Reply:
For Joy's points, one thing we could maybe do is to link to the glossary entries for change id and commit id from this question.

Reply:
I've added links now.